jenkins/cori_atmnbfb.sh

```diff
@@ -5,11 +5,4 @@ export SCRIPTROOT=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && cd .. && pwd )
 export CIME_MACHINE=cori-knl
 source $SCRIPTROOT/util/setup_common.sh
 
-# Unload python module from:
-# $SCRIPTROOT/util/setup_common.sh --> $SCRIPTROOT/util/${CIME_MACHINE}_setup.sh
-module unload python/2.7-anaconda-5.2
-
-# Get e3sm_simple conda env
-source /global/project/projectdirs/acme/software/anaconda_envs/load_latest_e3sm_simple.sh
```

Comment:
Should the corresponding `cime_env` script be getting sourced instead here?

Reply:
Or is the default python all that is needed for this test?

Reply:
This is specific to the `atmnbfb` job on cori, which is how we were previously sourcing it. Now, the `cori-knl_setup.sh` script loads the `cime_env` for all cori test jobs (haswell just links to the knl script) instead of the python module -- that is, we're loading it upstream now.

Reply:
Ah, got it.